evap/evaluation/migrations/0134_votetimestamp.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.1.5 on 2023-05-22 18:56
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0133_add_infotext_model'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="VoteTimestamp",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("timestamp", models.DateField(auto_now_add=True, verbose_name="vote timestamp")),
+                (
+                    "evaluation",
+                    models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to="evaluation.evaluation"),
+                ),
+            ],
+        ),
+        migrations.AlterField(
+            model_name="votetimestamp",
+            name="timestamp",
+            field=models.DateTimeField(auto_now_add=True, verbose_name="vote timestamp"),
+        ),
+        migrations.AlterField(
+            model_name="votetimestamp",
+            name="evaluation",
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to="evaluation.evaluation"),
+        ),
```

Comment:
Please combine these `AlterField` operations into the `CreateModel` operation above -- there's no need to create the model with the wrong field type and then immediately fix it in a separate operation.